client/blocks/comment-detail/style.scss

```diff
@@ -496,11 +499,19 @@ a.comment-detail__author-more-element {
 		padding-left: 8px;
 		text-overflow: ellipsis;
 	}
+
+	strong {
+		float: none;
```

Comment:
What does this do?

Reply:
The author block usually contains the author name and the author url, on the same line.
To work properly on `rtl` we added a `float: left` on the name (which is the `strong` there), that gets reversed as `float: right` on `rtl`.

Though, in the User Info block, the author block contains the author display name and the author username (when provided) on **two lines**.
The `float` basically places the display name and the username in the same position, and the former covers the latter.